roles/ceph-nfs/templates/ganesha.conf.j2

```diff
@@ -30,7 +30,7 @@ RADOS_URLS {
    ceph_conf = '/etc/ceph/{{ cluster }}.conf';
    userid = "{{ ceph_nfs_ceph_user }}";
 }
-%url rados://{{ cephfs_data_pool.name }}/{{ ceph_nfs_rados_export_index }}
+%url rados://{{ cephfs_data_pool.name }}/conf-{{ ansible_hostname }}
 
 NFSv4 {
 	RecoveryBackend = 'rados_kv';
```

Comment:
`watch_url:` is needed pointing to the `conf-<id>` object. See [reference](https://documentation.suse.com/ses/6/html/ses-all/cha-ceph-nfsganesha.html#ceph-nfsganesha-config-service-rados).

Reply:
What's the difference between `%url` outside of a configuration block, and using `watch_url` inside the `RADOS_URLS` section? I've had a look at the configuration reference but I can't quite see how they're used differently. Purely guessing, is `watch_url` used to trigger a reload if the configuration object changes, so that the the configuration loaded by the `%url` section is picked up?